samples/default-authorizationserver/src/test/java/sample/DefaultAuthorizationServerApplicationTests.java

```diff
@@ -46,15 +50,18 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @AutoConfigureMockMvc
 public class DefaultAuthorizationServerApplicationTests {
-	private static final String REDIRECT_URI = "http://127.0.0.1:8080/login/oauth2/code/messaging-client-oidc";
+	private static final String REDIRECT_URI = "http://127.0.0.1:8080/login/oauth2/code/messaging-client-oidc?param=is%2Fencoded";
+	public static final String STATE = "some+%20encoded%2Fstate";
+	public static final String REENCODED_STATE = "some%20%20encoded%2Fstate";
 
 	private static final String AUTHORIZATION_REQUEST = UriComponentsBuilder
 			.fromPath("/oauth2/authorize")
 			.queryParam("response_type", "code")
 			.queryParam("client_id", "messaging-client")
 			.queryParam("scope", "openid")
-			.queryParam("state", "some-state")
-			.queryParam("redirect_uri", REDIRECT_URI)
+			.queryParam("state", STATE)
```

Comment:
Please remove all changes related to the `state` parameter since this was fixed in gh-875 and test coverage is included in the associated commit